webpack.config.js

```diff
@@ -18,7 +19,14 @@ module.exports = {
         options: {
           configFile: "tsconfig.other.json"
         }
+      },
+      {
+        // Preprocess 3rd party .css files located in node_modules
+        test: /\.css$/,
+        include: /node_modules/,
```

Comment:
https://github.com/jeongm-in/webmark/issues/75#issuecomment-509733761
and the empty line below